### PR TITLE
Azure VM installation

### PR DESCRIPTION
**Pull request by TPGamesNL**
This PRs adds the ability to automatically turn this software into an Azure VM Application, which can then be used by the main Judge (queuer) to automatically install it on the VMs it creates.

VM Applications are versioned, which is implemented by Git tagging: whenever you push a tag to this repository, it'll automatically deploy it into a VM Application.

### CODE REVIEW

File: entrypoint.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+import sys
+import time
+
+if __name__ == "__main__":
+    print("Hello there")
+    time.sleep(10)
+    sys.exit(123)
```
Comment (andreitudorp) [inline entrypoint.py:8]: Is the 123 exit code intentional?

Code 123: The filename, directory name, or volume label syntax is incorrect.

**Review by andreitudorp — COMMENTED** [review]
One small comment (:

File: entrypoint.py (hunk 1 above)
Comment (TPGamesNL) [reply entrypoint.py:8]: This whole file is for testing :)

But I'll add a note about that

**Review by andreitudorp — APPROVED** [review]
CrazyEyesSteveBuscemiGIF

**Pull request merged**